src/compressed_tensors/quantization/quant_args.py

```diff
@@ -153,7 +169,7 @@ class QuantizationArgs(BaseModel, use_enum_values=True):
     group_size: Optional[int] = None
     strategy: Optional[QuantizationStrategy] = None
     block_structure: Optional[str] = None
-    dynamic: bool = False
+    dynamic: Optional[Union[DynamicType, bool]] = False
```

Comment:
So we want to allow for any type in `(True, False, DynamicType.Local, None)`? Why do we want to allow None type?

Reply:
The user can provide True, False or local.
If the user doesn't provide anything, we default to False.